package.json

```diff
@@ -26,10 +26,6 @@
     "watch": "rollup -c -w",
     "build:dist": "rimraf dist && node ./scripts/buildDist"
   },
-  "pre-commit": [
```

Comment:
you deleted my pre-commit :(

Reply:
thats weird. i definitely didn't manually do that. let me go get it back

Reply:
back and better than ever 😂